### PR TITLE
Add munnerz as a primary admin/approver

### PR DESCRIPTION
**Pull request by munnerz**
A while back at the last Slack admins meet, we discussed moving some non-primary admins to be 'primary' admins. I don't think this was ever put into effect though, so thought I'd open up a PR to add myself as an approver so I can help within this repo too 😄

There's also a Google Sheet that lists all the admins that'll need updating too, which I'll go take a look for after this has been reviewed/accepted 🙂

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
🎉 
/lgtm
/hold

**Comment by coderanger** [comment]
IIRC we need a +1 from a majority of existing approvers? I might be mixing that up with another process, but if so that means 7 +1s.

Adding mine, +1.
/lgtm

**Review by idvoretskyi — APPROVED** [review]
/lgtm

**Comment by mrbobbytables** [comment]
We've had enough acks in slack and here \o/

/hold cancel

**Pull request merged**
